readme.md

```diff
@@ -450,3 +461,36 @@ body, .foo {
   [RULE_END]  
 ]
 ```
+
+#### JavaScript Function Value
```

Comment:
I've written down the cases I'm aware of here :smile: https://gist.github.com/philpl/61e2d7ecfb03a1608931ca1942222947